packages/twenty-server/src/core/billing/billing.resolver.ts

```diff
@@ -38,11 +39,25 @@ export class BillingResolver {
     };
   }
 
-  @Mutation(() => CheckoutEntity)
+  @Query(() => SessionEntity)
   @UseGuards(JwtAuthGuard)
-  async checkout(
+  async billingPortalSession(
     @AuthUser() user: User,
-    @Args() { recurringInterval, successUrlPath }: CheckoutInput,
+    @Args() { returnUrlPath }: BillingSessionInput,
+  ) {
+    return {
+      url: await this.billingService.billingPortalSession(
```

Comment:
can we rename this method on billingService to computeBillingPortalSessionURL (same for computeCheckoutSessionURL)?